cmd/controller-manager/main.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2022 The SODA Authors.
```

Comment:
"Copyright 2022 The SODA Authors." -> "Copyright 2023 The SODA Authors."

Reply:
changed